src/index.html

```diff
@@ -5,10 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title>Flexbox</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="images/logo.png" alt="#"></a>
+      <nav>
+        <ul class="nav__lists">
+          <li class="nav__item">
+            <a href="#" class="nav__link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+            class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```

Comment:
apply code styling recommendations from checklist all over the page
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```